src/http/controllers/controller.dependents.ts

```diff
@@ -15,42 +11,27 @@ export module DependentsController {
      * @param res {Response}
      */
     export const _getDependents = async (req, res) => {
-        // validate permissions
-        const requestOwn = req.user.accountID === req.params.accountID;
-        const permissionsGranted =
-            requestOwn ?
-                ac.can(req.user.role).readOwn(PermissionActions.ACCOUNT)
-                :
-                ac.can(req.user.role).readAny(PermissionActions.ACCOUNT)
-            ;
-        if (permissionsGranted) {
-            const { accountID } =
-                req.user.role == PermissionRoles.USER ? req.user : req.params;
-            if (!accountID) {
+        const { accountID } =
+            req.user.role == PermissionRoles.USER ? req.user : req.params;
+        if (!accountID) {
```

Comment:
AccessControlMiddleware must grant only authenticated accounts runs this callback. We don't need verify if we get a accountID becouse we alwais will get it.

Reply:
Yes it's right only you need to use `req.params`. But are you sure dont you need to check if it's null?

Reply:
This is the problem I mentioned you that express didn't recognize the differente between `dependents` & `depdents/{id}`. If u want we can make a call to take more into detail about it.

Reply:
I split it in tow controllers.
Non ADMIN queries from myDependents/:dependentID/ uses _getMyDependentById in line 43. This controller returns one dependent  if a valid dependentID is provided or all account dependents if no dependentID is provided.
ADMIN quieries from  /dependentsFor/:accountID uses _getDependents in line 13.

Reply:
Please remove all this validations, it's not needed more due to our new implementation.